### ISSUE TITLE
Gtk: Bug with opening files

### ISSUE THREAD

**Issue by TheShadowOfHassen** [olivierkes/manuskript#1109]
This is just a heads up. @TheJackiMonster if your file doesn't have something in a certain pane Gtk manuskript won't display it.

For example in one of my test files I didn't have any files in the editor.(Or maybe even a selected one.) The entire editor pane didn't appear. I suppose this can be fixed when save is worked on a bit more but I just wanted to give a heads up.

**Comment by TheJackiMonster** [comment]
Yes, it's very much work-in-progress. I'm mostly testing whether the UI I'm working on acts similar as the Qt version of manuskript. So I don't test with empty projects for the most part. Once I encounter such an issue, I tend to fix it immediately. But these are special cases which have to be applied individually.

Since you are working on the GTK code now. If you have a project which breaks the applications logic, such behavior of the UI not appearing is typical. It's because the code of creating the UI elements breaks at one point but I catch the exception at some point I think. The goal is that the code creating UI elements can handle any data thrown at it, even if the project is somehow broken in itself.
